2022q2/lldb.adoc

```diff
@@ -0,0 +1,34 @@
+=== Ongoing work on LLDB multiprocess debugging support
+
+Links: +
+link:https://www.moritz.systems/blog/multiprocess-support-for-lldb/[Moritz Systems Project Description] URL: link:https://www.moritz.systems/blog/multiprocess-support-for-lldb/[https://www.moritz.systems/blog/multiprocess-support-for-lldb/] +
+link:https://www.moritz.systems/blog/implementing-non-stop-protocol-compatibility-in-lldb/[Progress Report 1] URL: link:https://www.moritz.systems/blog/implementing-non-stop-protocol-compatibility-in-lldb/[https://www.moritz.systems/blog/implementing-non-stop-protocol-compatibility-in-lldb/]
+
+Contact: Kamil Rytarowski <kamil@moritz.systems> +
+Contact: Michał Górny <mgorny@moritz.systems>
+
+According to the upstream description, "LLDB is a next generation,
+high-performance debugger. It is built as a set of reusable components which
+highly leverage existing libraries in the larger LLVM Project, such as the
+Clang expression parser and LLVM disassembler."
+
+FreeBSD includes LLDB in the base system.  At present, it has some limitations
+compared to the GNU GDB debugger, and does not yet provide a complete
```

Comment:
I wonder if we should soften this statement somewhat -- your previous work brought lldb to be a credible replacement to the GDB version that was in the base system.

Maybe something like "Previous sponsored projects improved LLDB, to make it a credible debugger for the base system, although it still has a few limitations compared to contemporary versions of GNU GDB."

Reply:
Thanks, that's a very good suggestion. I'll update the text shortly.